src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -81,6 +82,19 @@ const ChannelTitle = ({
 
   const isSignedDM = type === BaseChannelItemTypeProps.SIGNED_PM;
 
+  const getMessageText = () => {
+    if (hasAttachment) {
+      return (
+        <Text>
+          <Text>{isMe ? 'You: ' : ''}</Text>
+          <Text style={{fontStyle: 'italic'}}>Sent media 🏞️</Text>
+        </Text>
+      );
+    }
+
+    return `${isMe ? 'You: ' : ''}${message}`;
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getMessageText` is a good abstraction for handling the message text based on whether it has an attachment or not. However, I would suggest using template literals instead of string concatenation for better readability and performance.
```diff
- return `${isMe ? 'You: ' : ''}${message}`;
+ return isMe ? `You: ${message}` : message;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->